src/Mappers/CustomerMapper.php

```diff
@@ -88,8 +88,34 @@ public static function map(Response $response)
                     $customer->$method($value);
                 }
             }
+
+            $collectMandateElement = $responseDOM->getElementsByTagName('collectmandate')->item(0);
+
+            if ($collectMandateElement) {
+
+                // Collect mandate elements and their methods
+                $collectMandateTags = array(
+                    'id' => 'setID',
+                    'signaturedate' => 'setSignatureDateFromString',
+                    'firstrundate' => 'setFirstRunDateFromString',
+                );
+
+                $customer->setCollectMandate(new \PhpTwinfield\CustomerCollectMandate());
+
+                // Go through each collect mandate element and add to the assigned method
+                foreach ($collectMandateTags as $tag => $method) {
```

Comment:
Any of the convenience methods in `\PhpTwinfield\Mappers\BaseMapper` that can be useful here?